testing/regress/hpcc/common/logger.py

```diff
@@ -115,7 +115,7 @@ def emit(self, record):
                 msg = self.format(record)
                 stream = self.stream
                 isBuffer = hasattr(record, 'filebuffer')
-                toSort = hasattr(record,  'filesort')
+                #toSort = hasattr(record,  'filesort')
```

Comment:
Why is this commented out?

Reply:
That was an unused variable, removed. 
(Long time ago there was a forced sort for log lines when parallel queries used. It is already improved, the sort replaced by speed effective data structure, but somehow this line left there. Now the Python IDE put a notification to that line, I commented out before test and forget to remove before push.)